### PR TITLE
MM-49786 - add no thanks button to onboarding completed screen

### PR DESCRIPTION
**Pull request by pvev**
#### Summary
This PR adds the no thanks button to the onboarding completed screen

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-49786

#### Screenshots

https://user-images.githubusercontent.com/10082627/228473643-3c6fdddf-f490-4efb-a860-9957c51e44d6.mov

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by marianunez — APPROVED** [review]
Thanks for the quick fix @pvev!

File: webapp/channels/src/components/onboarding_tasklist/onboarding_tasklist_completed.tsx [hunk 1]
```diff
@@ -193,6 +211,15 @@ const Completed = (props: Props): JSX.Element => {
                                     onClick={dismissAction}
                                 />
                             )}
+                            <button
+                                onClick={dismissAction}
+                                className={'no-thanks-link style-link'}
+                            >
+                                <FormattedMessage
+                                    id={'onboardingTask.checklist.no_tanks'}
```
Comment (julmondragon) [inline webapp/channels/src/components/onboarding_tasklist/onboarding_tasklist_completed.tsx:219]: nit: `'onboardingTask.checklist.no_thanks'`

**Review by julmondragon — APPROVED** [review]
Thanks @pvev !

**Comment by pvev** [comment]
@annelieseklein gentle ping on this one :)

**Review by annelieseklein — APPROVED** [review]
Thanks!

**Pull request merged**